packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md

```diff
@@ -0,0 +1,58 @@
+| `PROXY_BUILD_PATH`                              | Your XM Cloud Proxy server path is needed to build the app. The build output will be copied to the proxy server path.
+| `SITECORE_EDGE_CONTEXT_ID`                              | The Context ID, which covers many system configurations, required for connecting to the XM Cloud back end. This is an XM Cloud system environment variable. When the application runs on the XM Cloud rendering host, this value is always set to the preview Context ID.                   |
+| `SITECORE_API_KEY`                              | The Context ID, which covers many system configurations, required for connecting to the XM Cloud back end. This is an XM Cloud system environment variable. When the application runs on the XM Cloud rendering host, this value is always set to the preview Context ID.                   |
+| `SITECORE_API_HOST`                              | The API hostname, needed to build the application. When the application runs on the XM Cloud rendering host, this value is always set to your content management instance, and it will override any value set in your .env file. For example, https://<xmc_cm_host>.sitecorecloud.io.                   |
+| `GRAPH_QL_ENDPOINT`                              | The GraphQL endpoint. For Experience Edge, the value is https://edge.sitecorecloud.io/api/graphql/v1. For a preview endpoint (a CM instance on XM Cloud or a local one), the value is <xmc_cm_host>/sitecore/api/graph/edge. When the application runs on the XM Cloud rendering host, this value is always set to the preview endpoint for your CM instance <xmc_cm_host>/sitecore/api/graph/edge , and it will override any value set in your .env file.    |
+| `SITECORE_SITE_NAME`                              | The name of your site. This variable overrides the config.appName defined in the package.json file. You can find this value in the Sites dashboard by opening the actions menu for a site and navigating to Settings > Developer settings.                   |
+| `DEFAULT_LANGUAGE`                              | The default language of your app.                   |
+| `FETCH_WITH`                              | Determines how layout and dictionary data is fetched from Sitecore. The possible values are REST and GraphQL. You select the value during the app scaffold process. If the value isn't provided, the default is GraphQL.                   |
+| `DEBUG`                  | Optional. Debug level for the proxy. Set the DEBUG environment variable to 'sitecore-jss:*,proxy*,http-proxy-middleware*' to see all logs.
+
+## Build & run
+
+Build your Angular SPA app bundle with `jss build`. The build output should be placed in the `dist` folder and will automatically get copied to the proxy app.
```

Comment:
I'm not sure here, but do you think it makes sense to list both `jss build` and `npm run build` commands? Either works, but it might be useful to see both options for people new to JSS

Reply:
I agree. Added.